gwcli/gateway.py

```diff
@@ -410,6 +410,56 @@ def ui_command_info(self):
                 console_message("- {} : {}".format(dict_key,
                                                    self.control_values[dict_key]))
 
+    def ui_command_reconfigure(self, attribute, value):
+        """
+        The reconfigure command allows you to tune various gatway attributes.
+        An empty value for an attribute resets the lun attribute to its
+        default.
+        attribute : attribute to reconfigure. supported attributes:
+          - cmdsn_depth : integer
+          - dataout_timeout : integer
+          - nopin_response_timeout : integer
+          - nopin_timeout : integer
+          - immediate_data : [Yes|No]
+          - initial_r2t : [Yes|No]
+          - first_burst_length : integer
+          - max_burst_length : integer
+          - max_outstanding_r2t : integer
+          - max_recv_data_segment_length : integer
+          - max_xmit_data_segment_length : integer
+        value     : value of the attribute to reconfigure
+        e.g.
+        set cmdsn_depth
+          - reconfigure attribute=cmdsn_depth value=128
+        reset cmdsn_depth
+          - reconfigure attribute=cmdsn_depth value=
+        """
+        if not attribute in settings.Settings.GATEWAY_SETTINGS:
+            self.logger.error("supported attributes: {}".format(",".join(
+                sorted(settings.Settings.GATEWAY_SETTINGS))))
+            return
+
+        # Issue the api request for the resize
```

Comment:
I think change "resize" to "reconfigure".

Reply:
Updated

Reply:
Updated